<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>832</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>사물함 반납</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="returnBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>사물함 반납하기</string>
   </property>
   <widget class="QLabel" name="name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>학생 이름 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="locker_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nanum Gothic</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>대여중인 사물함 : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="returnBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>반납하기</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="usageHistoryBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>361</width>
     <height>371</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nanum Gothic</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>사물함 이용 내역 확인하기</string>
   </property>
   <widget class="QPushButton" name="usageHistoryBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>확인하기</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QListWidget" name="usageHistoryList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>301</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
